api/definition/swagger.yaml

```diff
@@ -146,22 +147,19 @@ components:
           description: The year that the data point is for
         count:
           type: number
-          format: int32
           example: 222
           description: The count
         value:
           type: number
-          format: int32
           example: 506.60912
           description: The value
         lowerCi:
           type: number
-          format: int32
           example: 441.69151
           description: The lower confidence interval
         upperCi:
           type: number
-          format: int32
+          format: float
```

Comment:
This is a bit inconsistent now. `count`, `value` & `lowerCi` don't specify a format, but this one sets it to `float`.